plenum/server/node.py

```diff
@@ -596,6 +596,8 @@ def on_view_change_complete(self):
         if self.view_changer.propagate_primary:  # TODO VCH
             self.master_replica.on_propagate_primary_done()
         self.view_changer.last_completed_view_no = self.view_changer.view_no
+        for replica in self.replicas:
```

Comment:
Please add a comment on why we do it

Reply:
Should we describe the case with propagates or commented what we do?

Reply:
Yes, please describe the case with propagates (a reference to the tests would also be useful)